src/internal/pachd/setup.go

```diff
@@ -164,6 +166,18 @@ func initPFSAPIServer(out *pfs.APIServer, outMaster **pfs_server.Master, env fun
 		},
 	}
 }
+
+func initPJSAPIServer(out *pjs.APIServer, env func() pjs_server.Env) setupStep {
+	return setupStep{
+		Name: "initPFSAPIServer",
+		Fn: func(ctx context.Context) error {
+			apiServer := pjs_server.NewAPIServer(env())
+			*out = apiServer
```

Comment:
Why not `*out = pjs_server.NewAPIServer(env())`?

At some point, we should factor out these out-arguments …

Reply:
I prefer two lines since its more refactorable. One day NewAPIServer() might return an error, at which case this code will have to change less.